### PR TITLE
Sixpack powering up

### PR DESCRIPTION
**Pull request by weerd**
This PR adds a simple Sixpack A/B test to the campaign, switching out the some of the styling for the Competition block. This is mostly a test to enable a first test and confirm that Phoenix-Next-Thor is connecting properly to Sixpack-Thor.

This setup is not ideal since adding a test requires modifying too many different locations. I'm considering some alternatives for how best to structure A/B test so it will require modifying less files to execute on a test. However, these are the "pieces" needed to perform a full A/B test.

### CODE REVIEW

File: resources/assets/components/CompetitionBlock/competitionBlock.scss [hunk 1]
```diff
@@ -28,3 +28,16 @@
...
+
+.block {
+  &.-colorful {
+    background-color: $purple;
+    border-radius: 4px;
+  }
+}
+
+.competition-block {
+  .-colorful & {
```
Comment (DFurnes) [inline resources/assets/components/CompetitionBlock/competitionBlock.scss]: Absolutely not the point of this pull request, but this feels awkward – both modifying `.block` from the "competition block" file, and having to account for `.-colorful` modifier on both.

File: resources/assets/components/CompetitionBlock/index.js [hunk 2]
```diff
@@ -19,6 +20,20 @@ const CompetitionBlock = (props) => {
   const { content, photo, byline, joinCompetition, hasJoinedCompetition,
     hasPendingJoin, showConfirmation, campaignId, campaignRunId, checkForCompetition } = props;
 
+  // @TEST: Sixpack Experiments
```
Comment (DFurnes) [inline resources/assets/components/CompetitionBlock/index.js]: I like this! `@TEST:` 💡

File: resources/assets/components/CompetitionBlock/competitionBlock.scss (hunk 1 above)
Comment (weerd) [inline resources/assets/components/CompetitionBlock/competitionBlock.scss]: Yeah, this is by no means ideal. It's what I mention in the description with "This setup is not ideal..." 😉 I just wanted to get something up to test Sixpack on Thor and confirm it's working. Definitely need to spend some time on considering a better approach overall.

Comment (DFurnes) [inline resources/assets/components/CompetitionBlock/competitionBlock.scss]: I know, I know! Definitely not something to deal with in this PR. 😄

**Pull request merged**